extensions/sql-migration/src/api/sqlUtils.ts

```diff
@@ -394,11 +452,13 @@ export async function collectTargetLogins(
 	password: string,
 	includeWindowsAuth: boolean = true): Promise<string[]> {
 
-	const connectionProfile = getConnectionProfile(
+	const connectionProfile = getTargetConnectionProfile(
 		serverName,
 		azureResourceId,
 		userName,
 		password,
+		// for login migration, when connecting to a target Azure SQL, use true/true
```

Comment:
for login migration scenario, verify that true/true is correct for the target (regardless of target platform)

Reply:
cc @AkshayMata

Reply:
See comment above. To recap here the long term solution is:
- For SQL MI, default to true/false. 
- For SQL VM, we give customer UX dropdowns to select how they want to connect.